leaf-ui/rappid-extensions/BackboneModels.js

```diff
@@ -1 +1,27 @@
-/** This file contains backbone model representations of the original model objects - WIP */
\ No newline at end of file
+/** This file contains backbone model representations of the original model objects - WIP */
+
+/** 
+ * Backbone for UserEvaluation 
+ * */ 
+var UserEvaluationModel = Backbone.Model.extend({  
+    //Take in properties
+    initialize: function(options){ //Named arguments and passes in parameter
+        this.intentionID = options.intentionID; 
+        this.absTime = options.absTime;
+        this.evaluationValue = options.evaluationValue;
+    }
+});
+
+/** 
+ * Backbone for Constraint 
+ * */
+var ConstraintModel = Backbone.Model.extend({
+    initialize: function(options){ //Named arguments and passes in parameter
+        this.constraintType = options.constraintType;
+        this.constraintSrcID = options.constraintSrcID;
+        this.constraintSrcEB = options.constraintSrcEB;
```

Comment:
We can remove constraint from each of these. Then write out `srcID`. 
We are going to change the `EB` to be called `RefTP`, for Reference Time Point, as in `srcRefTP`.